contrib/gitian-keys/keys.txt

```diff
@@ -0,0 +1,27 @@
+152812300785C96444D3334D17565732E08E5E41 Andrew Chow
```

Comment:
Could you add my key fingerprints?
617C90010B3BD370B0AC7D424BB42E31C79111B8 Akira Takizawa

Reply:
Removal and addition of keys will happen in separate pull requests. You should wait until this one is merged.  
Might want to help by review, though.

Reply:
@akx20000a This has been merged, you can submit your PR now